rules_ruby_gapic/BUILD.bazel

```diff
@@ -36,7 +36,7 @@ ruby_binary (
 #
 ruby_binary (
   name = "gapic_generator_ads",
-  entrypoint = "@gapic_generator_src//:gen_src/gapic-generator-ads/bin/protoc-gen-ruby_ads",
+  entrypoint = "@gapic_generator_src//:gen_src/gapic-generator-ads/bin/protoc-gen-bazel-ruby_ads",
```

Comment:
Might be nicer to use `protoc-gen-bazel_ruby_ads` since this coresponds to a protoc parameter of `--bazel_ruby_ads_out` (instead of `--bazel-ruby_ads_out`)

Reply:
that's a good point